pkg/skuba/actions/cluster/init/init.go

```diff
@@ -41,6 +41,9 @@ import (
 	"github.com/SUSE/skuba/pkg/skuba"
 )
 
+// Basic initial cluster configuration
+// Note: UseHyperKube can be removed when we drop the support of
+// provisioning clusters of version 1.17.
```

Comment:
nit: recommend adding the comments above or after `UseHyperKube      bool`

Reply:
If I move it, it will be hidden in plain sight. I put it there so that people (or simply put, me ... :p) see it in their IDE.

But if folks wants to move it, I will bow to the common decision.